flow/controllers/car_following_models.py

```diff
@@ -56,15 +56,18 @@ def __init__(self,
                  v_des=8,
                  time_delay=0.0,
                  noise=0,
-                 fail_safe=None):
+                 fail_safe=None,
+                 display_warnings=True):
```

Comment:
Do these display a crazy amount of warnings? If so we might want to tset it to False